modules/create-new-system-policy.adoc

```diff
@@ -57,9 +57,9 @@ Select *ON* to enforce policy and report a violation. Select *OFF* to only repor
 ====
 The enforcement behavior is different for each lifecycle stage.
 
-* For the *Build* stage, {product-title} fails your CI builds when images match the conditions of the policy.
-* For the *Deploy* stage, {product-title} blocks the creation of deployments that match the conditions of the policy. In clusters with admission controller enforcement, the Kubernetes or {ocp} API server blocks all noncompliant deployments. In other clusters, {product-title} edits noncompliant deployments to prevent pods from being scheduled.
-* For the *Runtime* stage, {product-title} stops all pods that match the conditions of the policy.
+* For the *Build* stage, {product-title-short} fails your CI builds when images match the conditions of the policy.
+* For the *Deploy* stage, {product-title-short} blocks the creation and update of deployments that match the conditions of the policy if the {product-title-short} admission controller is configured and running. In clusters with admission controller enforcement, the Kubernetes or {ocp} API server blocks all noncompliant deployments. In other clusters, {product-title-short} edits noncompliant deployments to prevent pods from being scheduled. For existing deployments, policy changes only result in enforcement at the next detection of the criteria, when a Kubernetes event occurs. For more information about enforcement, see "Security policy enforcement for the deploy stage".
```

Comment:
Add a sub-bulleted point to the second bullet point to reduce the larger paragraph block?